scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -447,13 +585,14 @@ private static void storeGroups(String groupFilePath, Multimap<String, String> g
         } catch (IOException e) {
             exitWithErrorMessage("could not write group file : " + groupFilePath, null, e);
         }
+        System.out.println("Stored group file in " + groupFilePath);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS106)